praw/models/reddit/collections.py

```diff
@@ -457,7 +461,7 @@ def create(self, title: str, description: str):
         .. code-block:: python
 
             my_sub = reddit.subreddit("test")
-            new_collection = my_sub.collections.mod.create("Title", "desc")
+            new_collection = my_sub.collections.mod.create("Title", "desc", "GALLERY")
```

Comment:
```suggestion
            new_collection = my_sub.collections.mod.create("Title", "desc")
```
Lets make a separate example for this.

Reply:
Ah, from the previous point about empty argument. Got it.